app/src/main.ts

```diff
@@ -169,7 +169,7 @@ if (appArgs.lang) {
 let currentBadgeCount = 0;
 const setDockBadge = isOSX()
   ? (count?: number | string, bounce = false): void => {
-      if (count) {
+      if (typeof count !== 'undefined') {
```

Comment:
This works, but out of curiosity: any reason to not simply

```suggestion
      if (count !== undefined) {
```

Reply:
Good point. Given it's a function argument, it makes sense.